data_construction/EMP_data_construction/format_metadata.R

```diff
@@ -0,0 +1,92 @@
+  data.list[[i]] <- subset(data.list[[i]], data.list[[i]]$sample_name %in% samples)
+}
+
+#make column names consistent
+for(i in 1:length(data.list)){
+    colnames(data.list[[i]])[colnames(data.list[[i]]) == 'carb_nitro_ratio'] <- 'c_n_ratio'
+    colnames(data.list[[i]])[colnames(data.list[[i]]) == 'perc_nitrogen'] <- 'percent_n'
+    colnames(data.list[[i]])[colnames(data.list[[i]]) == 'perc_total_c'] <- 'percent_c'
+    if('tot_carb' %in% colnames(data.list[[i]]) & 'tot_org_carb' %in% colnames(data.list[[i]])){
+      data.list[[i]] <- data.list[[i]][-data.list[[i]]$tot_org_carb]
+    }
+    if('tot_carb' %in% colnames(data.list[[i]]) & 'percent_c' %in% colnames(data.list[[i]])){
+      data.list[[i]] <- data.list[[i]][-data.list[[i]]$tot_carb]
+    }
+}
+
+#make units consistent
+data.list[[1]]$tot_nitro     <- data.list[[1]]$tot_nitro / 1000 * 100 # g N / kg -> kg N / kg soil -> percent N
+data.list[[1]]$tot_org_carb  <- data.list[[1]]$tot_org_carb / 1000 * 100 # g C / kg -> kg C / kg soil -> percent C
+colnames(data.list[[1]])[colnames(data.list[[1]]) == "tot_nitro"]      <- "percent_n"
+colnames(data.list[[1]])[colnames(data.list[[1]]) == "tot_org_carb"]   <- "percent_org_c"
+#3.   808: water content: ???
+data.list[[3]]$tot_carb      <- data.list[[3]]$tot_carb / 1000 * 100 
+data.list[[3]]$tot_nitro     <- as.numeric(data.list[[3]]$tot_nitro) / 1000 * 100 ## Warning: / not meaningful for factors
+colnames(data.list[[3]])[colnames(data.list[[3]]) == "tot_nitro"]      <- "percent_n"
+colnames(data.list[[3]])[colnames(data.list[[3]]) == "tot_carb"]       <- "percent_c"
+data.list[[4]]$tot_nitro     <- as.numeric(data.list[[4]]$tot_nitro) / 1000 * 100 ## Warning: / not meaningful for factors
+colnames(data.list[[4]])[colnames(data.list[[4]]) == "tot_nitro"]      <- "percent_n"
+data.list[[6]]$tot_nitro     <- data.list[[6]]$tot_nitro / 1000 * 100
+data.list[[6]]$tot_org_carb  <- data.list[[6]]$tot_org_carb / 1000 * 100
+colnames(data.list[[6]])[colnames(data.list[[6]]) == "tot_nitro"]      <- "percent_n"
+colnames(data.list[[6]])[colnames(data.list[[6]]) == "tot_org_carb"]   <- "percent_org_c"
+data.list[[7]]$tot_nitro     <- data.list[[7]]$tot_nitro / 1000 * 100
+data.list[[7]]$tot_org_carb  <- data.list[[7]]$tot_org_carb / 1000 * 100
+colnames(data.list[[7]])[colnames(data.list[[7]]) == "tot_nitro"]      <- "percent_n"
+colnames(data.list[[7]])[colnames(data.list[[7]]) == "tot_org_carb"]   <- "percent_org_c"
+#11. 1579: water content: ???
+data.list[[13]]$tot_nitro    <- data.list[[13]]$tot_nitro / 1000 * 100
+data.list[[13]]$tot_org_carb <- data.list[[13]]$tot_org_carb / 1000 * 100
+colnames(data.list[[13]])[colnames(data.list[[13]]) == "tot_nitro"]    <- "percent_n"
+colnames(data.list[[13]])[colnames(data.list[[13]]) == "tot_org_carb"] <- "percent_org_c"
+data.list[[14]]$tot_nitro    <- data.list[[14]]$tot_nitro / 1000 * 100
+colnames(data.list[[14]])[colnames(data.list[[14]]) == "tot_nitro"]    <- "percent_n"
```

Comment:
surprised these weren't caught in your loop call to convert everything with this column name to a different name? That would be better. Less code, easier to read, will generalize to more datasets in the event that happens.